Solutions/MimecastSEG/Data Connectors/MimecastSEG_API_AzureFunctionApp.json

```diff
@@ -0,0 +1,153 @@
+      "instructions": [
+        {
+          "parameters": {
+            "fillWith": [
+              "WorkspaceId"
+            ],
+            "label": "Workspace ID"
+          },
+          "type": "CopyableLabel"
+        },
+        {
+          "parameters": {
+            "fillWith": [
+              "PrimaryKey"
+            ],
+            "label": "Primary Key"
+          },
+          "type": "CopyableLabel"
+        }
+      ]
+    },
+    {
+      "title": "Deploy the Mimecast Secure Email Gateway Data Connector:",
+      "description": "\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinel-MimecastSEG-azuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n3. Enter the following fields:\n - appName: Unique string that will be used as id for the app in Azure platform\n - objectId: Azure portal ---> Azure Active Directory ---> more info ---> Profile -----> Object ID\n - appInsightsLocation(default): westeurope\n - mimecastEmail: Email address of dedicated user for this integraion\n - mimecastPassword: Password for dedicated user\n - mimecastAppId: Application Id from the Azure Sentinel app registered with Mimecast\n - mimecastAppKey: Application Key from the Azure Sentinel app registered with Mimecast\n - mimecastAccessKey: Access Key for the dedicated Mimecast user\n - mimecastSecretKey: Secret Key for dedicated Mimecast user\n - mimecastBaseURL: Regional Mimecast API Base URL\n - activeDirectoryAppId: Azure portal ---> App registrations ---> [your_app] ---> Application ID\n - activeDirectoryAppSecret: Azure portal ---> App registrations ---> [your_app] ---> Certificates & secrets ---> [your_app_secret]\n\n >Note: If using Azure Key Vault secrets for any of the values above, use the`@Microsoft.KeyVault(SecretUri={Security Identifier})`schema in place of the string values. Refer to [Key Vault references documentation](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) for further details.\n\n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy.\n\n6. Go to ***Azure portal ---> Resource groups ---> [your_resource_group] --->  [appName](type: Storage account) ---> Storage Explorer ---> BLOB CONTAINERS ---> SIEM checkpoints ---> Upload*** and create empty file on your machine named checkpoint.txt and select it for upload (this is done so that date_range for SIEM logs is stored in consistent state)\n"
+    }
+  ],
+  "metadata": {
+    "id": "Unique Identifier (GUID) used to identify dependencies and content from solutions or community.",
```

Comment:
The value for id is not mentioned, please check and update the value for id with proper GUID, thanks.

Reply:
Thanks for the info @v-sabiraj , please clarify where/how can I get the GUID. Thanks in advance.

Reply:
@v-sabiraj also, please let me know if there is some convention that needs to be adhered to, if we have to provide the mentioned GUID so that it is provided in a proper manner. Thanks!

Reply:
@v-sabiraj Please clarify if the GUID should be provided/generated by ourselves and hardcoded, or this should be handled in some other manner.

Reply:
@nkarisic, GUID should be provided by you. It can be any simple guid, e.g **"b9b685f7-31d8-4174-ad25-ee6ba0bd299a"**, thanks.

Reply:
@v-sabiraj Ok, thanks for the input, I'll add it to all of the connectors.